assign-doi-datacite/src/test/java/no/unit/nva/doi/datacite/config/DataCiteConfigurationFactoryTest.java

```diff
@@ -44,6 +54,19 @@ void setUp() {
         setupSystemUnderTest();
     }
 
+    @Test
+    void constructorWithExampleConfigAsInputstreamThenContains2KnownCustomers()
+        throws DataCiteMdsConfigValidationFailedException {
+        sut = createDataCiteConfigurationFactoryFromInputStream();
```

Comment:
Still confused by "sut"

Reply:
I guess it means "system under test" but I would prefer another name too.

Reply:
changed this into `dataCiteConfigurationFactory`.